features/display-contents.yml

```diff
@@ -0,0 +1,8 @@
+name: "display: contents"
+description: "The `display: contents` CSS declaration sets an element to display only its contents, not itself. The element does not generate a display box and may lose semantic or accesibility features."
+spec: https://drafts.csswg.org/css-display-3/#box-generation
+# caniuse: css-display-contents
```

Comment:
Disagreement between Can I Use and BCD regarding level of support (versions match)